addons/website/views/snippets/s_image_frame.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_image_frame" name="Image - Frame">
+    <section class="s_image_frame o_cc o_cc2 pt64 pb64">
```

Comment:
Suggestion but I'd switch to `o_cc_1`. The main point is that due to the padding applied on the section, it looks buggy on mobile.

<img width="200" alt="image" src="https://github.com/user-attachments/assets/d58050f2-ae4b-4707-9ade-90795f900b8f">